### PR TITLE
Updated README and addition of run_backpack.sh script

### PR DESCRIPTION
**Pull request by dry923**
Updated README and added a run_backpack.sh script that will make it easier to run outside of ripsaw

### CODE REVIEW

File: run_backpack.sh [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+LABEL_NAME=""
+LABEL_VALUE=""
+
+while getopts s:p:n:xa:t:c:u:l:v:h flag
+do
+    case "${flag}" in
+        s) ES_SERVER=${OPTARG};;
+        p) ES_PORT=${OPTARG};;
+        n) NAMESPACE=${OPTARG};;
+        x) PRIVILEGED=true;;
```
Comment (rsevilla87) [inline run_backpack.sh:22]: It would be great to have an IMAGE argument to customize the backpack image

Comment (dry923) [reply run_backpack.sh:22]: I like that idea. updated

**Review by jtaleric — APPROVED** [review]
LGTM!

**Review by rsevilla87 — APPROVED** [review]
LGTM!

**Pull request merged**